### ISSUE TITLE
List of working user agents for sites with strengthened Cloudflare protection

### ISSUE THREAD

**Issue by ViktorVoloshko** [tachiyomiorg/tachiyomi-extensions#17355]
### Describe why this should be added

Well, title says for itself, I think there should be something like wiki where users can post user agents that working for them for specific sources. Or maybe if we put them in something like .toml then someone can develop plugin that allows to fetch them and put directly in plugin settings without need to even open a browser. By the way, there what worked for me.

nhentai: ```Mozilla/5.0 (Linux; Android 13) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.5790.136 Mobile Safari/537.36```

All other sites working good with the default one.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by alessandrojean** [comment]
> someone can develop plugin that allows to fetch them and put directly in plugin settings without need to even open a browser

Extensions that heavily depends on it already does this by fetching a User Agent list from https://github.com/tachiyomiorg/user-agents.

> users can post user agents that working for them

This will only make the strings easier to be manually blocked by the site owners.
